components/widgets/ui/src/main/resources/PhenoTips/YesNoNAPicker.xml

```diff
@@ -277,7 +277,7 @@ $xwiki.ssx.use('PhenoTips.YesNoNAPicker')##
     var value = element.value;
     var selected = element.checked;
     var label = element.parentNode;
-    if (value != this.yesInput.value || !selected || !label) {
+    if (value != this.yesInput.value || !selected || !label || !value.startsWith("HP:")) {
```

Comment:
I don't like this, since it fixes PT-2391 just for the HPO. This condition seems to work, though:

     || (element.name !== '' && element.name !== this.yesInput.name)

Can you re-test it?

Reply:
@sdumitriu tested new fix, it fixes both original issue and regression